ideascube/serveradmin/catalog.py

```diff
@@ -701,13 +716,16 @@ def upgrade_packages(self, ids):
             except Exception as e:
                 printerr("Failed installing {0.id}\n".format(upkg))
                 printerr(e)
+                removed_and_not_upgraded_ids.append(upkg.id)
```

Comment:
This seems wrong when thinking of the UX: after an update, it is possible that the user won't see their cards on the home page. :confused: 

I don't think an update should change the setting.

Sure, it is possible that the update somehow fails to install the new version. But then the admin will fix the problem so that the new version ends up on the machine, and the setting should still have it.